packages/driver/cypress/integration/commands/actions/type_events_spec.js

```diff
@@ -0,0 +1,770 @@
+const { _, $ } = Cypress
+
+describe('src/cy/commands/actions/type - #type events', () => {
+  beforeEach(() => {
+    cy.visit('fixtures/dom.html')
+  })
+
+  describe('events', () => {
```

Comment:
Would it be appropriate to rename this to `keyboard events`?

Reply:
You're right. `keyboard events` better describes the tests. I fixed it that way.